BO4E/COM/Rechenschritt.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Text.Json.Serialization;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Ein Berechnungsschritt ist Teil einer <see cref="BO4E.BO.Berechnungsformel"/>
+    /// </summary>
+    /// <remarks>Entspricht der SG9 in UTILTS</remarks>
+    public class Rechenschritt : COM
+    {
+        /// <summary>
+        /// Die BestandteilID des Rechenschritt (1-99999)
+        /// </summary>
+        /// <remarks>UTILTS SG8 RFF 1154 / SG</remarks>
+        [JsonProperty(Required = Required.Always, PropertyName = "rechenschrittBestandteilId")]
+        [JsonPropertyName("rechenschrittBestandteilId")]
+        public int RechenschrittBestandteilId { get; set; }
+
+        /// <summary>
+        /// Die ReferenzID des Rechenschritt (1-99999)
+        /// </summary>
+        /// <remarks>UTILTS SG8 RFF 1154 / SG</remarks>
+        [JsonProperty(Required = Required.Always, PropertyName = "referenzRechensschrittID")]
+        [JsonPropertyName("referenzRechensschrittID")]
+        public int ReferenzRechenschrittId { get; set; }
+
+        /// <summary>
+        /// Die Rechenoperation
+        /// </summary>
+        [JsonProperty(Required = Required.Always, Order = 3, PropertyName = "operation")]
+        [JsonPropertyName("operation")]
+        [BoKey]
+        [ProtoMember(3)]
+        public ArithmetischeOperation Operation { get; set; }
+
+        /// <summary>
+        /// Ein möglicher Trafoverlust
+        /// </summary>
+        [JsonProperty(Required = Required.AllowNull, Order = 4, PropertyName = "verlustTrafo")]
+        [JsonPropertyName("verlustTrafo")]
+        [ProtoMember(4)]
+        public decimal? VerlustTrafo { get; set; }
```

Comment:
Können wir das mal mit der UTILMD verifizieren, ich glaube da heißt es VerlustfaktorTrafo

Reply:
In der Messlokation heißt es "Verlustfaktor"
https://github.com/Hochfrequenz/BO4E-dotnet/blob/c0cecfbad9f3db15a43fdb499511a4d984ebf6ea/BO4E/BO/Messlokation.cs#L204

ohen trafo oder leitung. aber ich werde es hier auch verlust*faktor* nennen

Reply:
e7f339d6100abe7a969b251f93cecadc7221e648